docker/initialization/genesis-maru.json.template

```diff
@@ -3,7 +3,12 @@
   "config": {
     "0": {
       "type": "delegated",
-      "blockTimeSeconds": 1
+      "blockTimeSeconds": 4,
```

Comment:
Why `blockTimeSeconds` = 4 instead of 1 here?

Reply:
There is no reason actually. And even for Maru, I think we've fixed an issue because of which we've set 7 seconds of block time. So I'll try to set it to 1 second and will see how it works on CI